index.d.ts

```diff
@@ -0,0 +1,98 @@
+// CREDITS: This types are based on the original work made by all the people who contributed to @types/react-native
+
+declare module "@react-native-community/status-bar" {
```

Comment:
You do not need declare module. Just place the status bar near source and properly map typings from package.json

Reply:
I know, but the other modules on `react-native-community` are using this approach (`react-native-maps`,  `react-native-localize`, `react-native-modal`, etc) so I thought it made sense to use it.

I don't think it is necessary to map the types as long as they are in the root of the package, we can add them though.

Quote from https://www.typescriptlang.org/ -> "Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so."

Thoughs?